lib/l10n/intl_fr.arb

```diff
@@ -0,0 +1,27 @@
+{
+  "@@locale": "fr",
+  "backButtonTooltip": "Retour",
+  "closeButtonLabel": "Fermer",
+  "searchLabel": "Rechercher",
+  "closeNavigationTooltip": "Fermer le navigateur",
+  "openNavigationTooltip": "Ouvrir le navigateur",
+  "clickToSearch": "Cliquez pour rechercher",
+  "modalBarrierDismissLabel": "Annuler",
+  "minimizeWindowTooltip": "Réduire",
+  "restoreWindowTooltip": "Restaurer",
+  "closeWindowTooltip": "Fermer",
+  "dialogLabel": "Dialogue",
+  "cutActionLabel": "Couper",
+  "copyActionLabel": "Copier",
+  "pasteActionLabel": "Coller",
+  "selectAllActionLabel": "Tout sélectionner",
+  "newTabLabel": "Ajouter un nouvel onglet",
+  "closeTabLabelSuffix": "Fermer l'onglet",
+  "scrollTabBackwardLabel": "Défilement vers l'arrière",
```

Comment:
`Défiler vers l'arrière` to match with the next item `Défiler vers l'avant`

Reply:
@WinXaito Solved!